build.sbt

```diff
@@ -4,12 +4,15 @@ import scala.concurrent.duration.FiniteDuration
 
 import sbt.enablePlugins
 
-Global / scalaVersion := "2.13.3"
-
```

Comment:
I don't think we need to remove this?

Reply:
It's not removed just moved down line no 15